tests/Unit/ApplicationTest.php

```diff
@@ -8,9 +8,9 @@ class ApplicationTest extends \PHPUnit_Framework_TestCase
 {
     const PROJECT_DIR = "/tmp/php-test-reporter-example-project";
 
-    protected $srcDir;
+    private $srcDir;
 
-    public function setUp()
+    protected function setUp()
```

Comment:
This is the visibility as declared on the parent, there's probably no need to increase it, is there?